lib/elixir/lib/enum.ex

```diff
@@ -276,10 +276,9 @@ defmodule Enum do
   end
 
   @doc """
-  Returns `true` if the given `fun` evaluates to a truthy value (neither `false` nor `nil`)
-  on all of the items in the `enumerable`.
-
-  It stops the iteration at the first invocation that returns either `false` or `nil`.
+  Iterates over the `enumerable` and invokes `fun` on each item. When an invocation
```

Comment:
We tend to keep the first paragraph of the documentation concise and simple, typically one-line. Tools like ExDoc use the first line to generate a summary 🙂 

(And don't forget to run the formatter `mix format`)

Reply:
Thanks!
Done and done.